roles/matrix-synapse/templates/synapse/homeserver.yaml.j2

```diff
@@ -276,7 +276,7 @@ listeners:
 
 {% if matrix_synapse_federation_port_enabled and matrix_synapse_tls_federation_listener_enabled %}
   # TLS-enabled listener: for when matrix traffic is sent directly to synapse.
-  - port: 8448
+  - port: {{ matrix_federation_public_port }}
```

Comment:
This seems to be confusing `matrix_federation_public_port` and `matrix_synapse_container_default_federation_api_port` (well, `matrix_synapse_container_federation_api_port` if the `_default` part of the name is dropped).

`matrix_federation_public_port` is the public port number. `matrix_synapse_container_federation_api_port` is the port inside the Synapse container. They don't really need to match.